pkg/cmd/pr/checks/checks.go

```diff
@@ -20,10 +20,23 @@ import (
 
 const defaultInterval time.Duration = 10 * time.Second
 
+var prCheckFields = []string{
+	"name",
+	"state",
+	"startedAt",
+	"completedAt",
+	"link",
+	"bucket",
```

Comment:
I know this is not the right place to ask this, but can't find the better one and https://cli.github.com/manual/gh_pr_checks is light on details.
What is the `bucket` field? Looking at the values it has, it's similar to status, right?

Reply:
FWIW, it's defined and filled in https://github.com/cli/cli/blob/e36d8f103d5e1fb3989d9d5aefdae36b01fb88d4/pkg/cmd/pr/checks/aggregate.go#L72-L88

It seems to be grouping checks by results

Reply:
@findepi : I think this is a great question and might be worth opening a new issue for improving the `--help` output to include a little more information about buckets.

Reply:
thanks @nobe4 for the link, the code suggests i want to use this field for filtering
thanks @andyfeller for suggestion -- https://github.com/cli/cli/issues/9205